Mlem/App/Views/Root/Tabs/Settings/SafetyBlurNsfwSettingsView.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SafetyBlurNsfwSettingsView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 2025-01-24.
+//
+
+import SwiftUI
+
+struct SafetyBlurNsfwSettingsView: View {
+    @Environment(Palette.self) var palette
+    @Setting(\.blurNsfw) var blurNsfw
+    
+    var body: some View {
+        Form {
+            headerView
+            Picker("Blur NSFW Content", selection: $blurNsfw) {
+                ForEach(NsfwBlurBehavior.allCases, id: \.self) { type in
+                    Label(String(localized: type.label), systemImage: type.systemImage)
+                }
+            }
+            .pickerStyle(.inline)
+            .labelsHidden()
+        }
+        .contentMargins(.top, 16)
+        .labelStyle(.conditional)
+    }
+    
+    @ViewBuilder
+    var headerView: some View {
+        SettingsHeaderView(
+            title: "Blur NSFW Content",
+            description: "Choose which contexts Not Safe For Work content should be blurred in.",
```

Comment:
*nit*

```suggestion
            description: "Choose when Not Safe For Work content should be blurred.",
```

Alternatively:
`Choose contexts in which to blur Not Safe For Work content`, if we want to keep the "in which contexts" phrasing--the basic revision here is to keep "in" and "which" together

Reply:
"When" and "where" are both good contenders here--I picked "when" because it feels slightly more natural ("When should I blur? -> Always/outside NSFW comm/never" vs "Where should I blur? -> Everywhere/outside NSFW comm/nowhere")